xonsh/completer.py

```diff
@@ -52,11 +55,22 @@ def startswithnorm(x, start, startlow=None):
 
 
 def _normpath(p):
-    # Prevent normpath() from removing initial ‘./’
-    here = os.curdir + os.sep
-    if p.startswith(here):
-        return os.path.join(os.curdir, os.path.normpath(p[len(here):]))
-    return os.path.normpath(p)
+    """ Wraps os.normpath() to avoid removing './' at the beginning 
+        and '/' at the end. On windows it returns a path with forward slashes
+    """   
+    initial_dotslash = p.startswith(os.curdir + os.sep)
+    initial_dotslash |= (os.altsep and p.startswith(os.curdir + os.altsep))
+    p = p.rstrip()
+    trailing_slash = p.endswith(os.sep) 
+    trailing_slash |= (os.altsep and p.endswith(os.altsep))
+    p = os.path.normpath(p)
+    if initial_dotslash:
+        p = os.path.join(os.curdir, p)
+    if trailing_slash:
+        p = os.path.join(p,'')
+#    if ON_WINDOWS:
+#        p = p.replace(os.sep, os.altsep)        
```

Comment:
Oh yes. Sorry @scopatz ..  I actually added them for a reason. Just forgot to mention it. I was wondering if it would be an idea to just use forward slashes even on Windows. It should off course accept both separators on windows, but only show forward slashes and convert everything to forward slashes when completing. 

There are of course pro/cons to this. Linux natives may find this nicer when they occasionally work on windows. Even someone like me who mostly use windows finds it much more convenient to use '/' when working in Python. The con is that it may not look as windows like... Who else from the development team uses windows sometimes ( maybe @rbrewer123 have an opinion on this)